roles/addons/nic_nmcli/readme.rst

```diff
@@ -127,22 +134,17 @@ You can define routes at two levels:
               mac: 08:00:27:36:c0:ac
               network: ice1-1
               routes4:
-                - ip = 10.11.0.0/24, nh = 10.10.0.2
-                - ip = 10.12.0.0/24, nh = 10.10.0.2
-
-.. note::
-  Note that to define a default route/gateway, use *0.0.0.0/0* as route to be defined.
-
-To remove a route later (here *10.12.0.0/24 10.10.0.2* on *enp0s8*), use the nmcli command this way:
-
-.. code-block:: text
+                - 10.11.0.0/24 10.10.0.2
+                - 10.12.0.0/24 10.10.0.2 300
 
-  nmcli connection show enp0s8 | grep ipv4.routes
-  nmcli connection modify enp0s8 -ipv4.routes "10.12.0.0/24 10.10.0.2"
+.. note:
+  In route4 list, first element is network to reach (0.0.0.0/0 for all), second
```

Comment:
Ambiguous. Each element of the list is a tuple with the network destination in first position, gateway in second position and optionally the metric in third position.

Reply:
Ok, I took your sentence.